src/config.js

```diff
@@ -4,3 +4,13 @@ export const BLOG_URL = process.env.REACT_APP_BLOG_URL;
 export const LOGIN_URL = process.env.REACT_APP_LOGIN_URL;
 export const REGISTER_URL = process.env.REACT_APP_REGISTER_URL;
 export const API_BASE_URL = process.env.REACT_APP_API_BASE_URL;
+//crane cloud status
```

Comment:
@neelxie great work, I think it would be better to be consistent and pick these URLs from the .env as well. This would help incase there is a change in any of them, case in point when using a custom domain!

Reply:
@e-ian This has been fixed sir.